skins/figures/index.ess

```diff
@@ -1,2 +1,56 @@
-display block
-margin 0 0 36px 0
\ No newline at end of file
+// initials
+border = '2px solid ' + __.color.gray
+border-radius 1000px
+color = __.color.gray
+cursor pointer
+position relative
+box-sizing border-box
+  
+&-content
+  font-weight bold
+  left 50%
+  margin 1px 0 0 0
+  position absolute
+  text-transform uppercase
+  top 50%
+  transform translate(-50%, -50%)
+
+// avatars
+.FiguresAvatar
```

Comment:
Yeah. There's cleanup here we're going to do. The default figure will be figure avatar. It's in a separate PR that Kevin and I worked on together. We'll keep working on keeping our branches only related to the skins we're working on.